querybook/webapp/components/QueryRunButton/QueryRunButton.tsx

```diff
@@ -93,6 +102,15 @@ export const QueryRunButton = React.forwardRef<
             />
         );
 
+        const tableSamplingDOM =
+            !disabled && TableSamplingConfig.enabled && hasSamplingTables ? (
+                <TableSamplingSelector
+                    sampleRate={sampleRate}
+                    setSampleRate={onSampleRateChange}
+                    tooltipPos={runButtonTooltipPos}
+                />
+            ) : null;
+
```

Comment:
can this live outside of QueryRunButton? I dont think it should control the UI for TableSamplingSelector, that's the job of the datacell or querycomposer

Reply:
will address it in a separate PR to use backend limit logic.